lib/main.js

```diff
@@ -44,6 +44,10 @@ module.exports = {
     return this.autocompleteManager
   },
 
+  provideManagerAsync () {
```

Comment:
Do you think we need to add the suffix `async` to the name of this method? Seems like we don't need it but I don't care *that* much.